src/lib/Scenes/Home/Components/Sales/Relay/SalesRenderer.tsx

```diff
@@ -0,0 +1,26 @@
+import createEnvironment from "lib/relay/createEnvironment"
+import React from "react"
+import { graphql, QueryRenderer } from "react-relay"
+import { PAGE_SIZE } from "../Components/LotsByFollowedArtists"
+
+export function SalesRenderer({ render }) {
+  return (
+    <QueryRenderer
+      environment={createEnvironment()}
```

Comment:
We'll want to use a shared environment instance here. Although there's some work being done in reaction to provide `environment` via the react context, I suggest we do the following in `lib/relay/createEnvironment`

```typescript
export var defaultEnvironment = createEnvironment()
```
and use it here.

Reply:
Ahh yes, let me do that...